README.md

```diff
@@ -228,7 +256,7 @@ A Basic configuration could look like this:
 DTU=ahoy
 
 #Possible Options for Log Level: CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET
-#To keep current.log small use ERROR
+#To keep current log small use ERROR or CRITICAL
 Logging=ERROR
```

Comment:
Currently we are not very straight about the used log levels in the, "CRITICAL" ist only used if startup fails, Error only for one config file check. Most of the important problems are Warnings. DEBUG is really for debugging.

So I'd say we recommend "WARNING" here to get helpful logs from the users. (Don't forget to update the comment)

Reply:
we also see get error codes in the log with CRITICAL and ERROR, so I think that this is still ok to use. and for more, we can go to warning, info and debug

Reply:
Yes, but I think if the default is Crit or Err, then too much helpful info will be lost in many cases.

Reply:
And it will not be too verbose when set to crit or err.

Reply:
we only want to have people to look into this once, they have a problem

we still see this level of detail: 

@4000000066543f070f950d44 Terminated
@4000000066543f0e276e2784 2024-05-27 08:06:28,660 root CRITICAL Start
@4000000066543f0e27f6e754 2024-05-27 08:06:28,669 root CRITICAL Registering dtu devices
@4000000066543f0e3386c1fc 2024-05-27 08:06:28,863 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 34
@4000000066543f0e3a84ef24 2024-05-27 08:06:28,980 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 35
@4000000066543f0f06e0eaec 2024-05-27 08:06:29,114 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 36
@4000000066543f0f0b05d894 2024-05-27 08:06:29,183 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 37
@4000000066543f0f106739f4 2024-05-27 08:06:29,273 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 38
@4000000066543f0f149aecdc 2024-05-27 08:06:29,344 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 39
@4000000066543f0f18dd2a6c 2024-05-27 08:06:29,416 root CRITICAL com.victronenergy.pvinverter /DeviceInstance = 40